airbyte-server/build.gradle

```diff
@@ -44,3 +44,9 @@ run {
     environment "DATABASE_PASSWORD", env.DATABASE_PASSWORD
     environment "DATABASE_URL", "jdbc:postgresql://localhost:5432/airbyte"
 }
+
+task copyOpenApiConfig(type: Copy) {
+    from file(rootProject.file("airbyte-api/src/main/openapi/config.yaml"))
```

Comment:
is there any way to package this as part of the artifact that `airbyte-api` creates? maybe instead of copying the file here, have it be a resoure in the jar of `airbyte-api`? not a huge fan of this copy of one source file to another source file.

if we do stick with this is there any special dependency that we need to declare here to say that this build depends on a source file (as opposed to the output) of a submodule?

Reply:
I wonder if we can define a gradle source set instead that points to this file .